### PR TITLE
Profile page

### PR DESCRIPTION
**Pull request by abhishektanwar**
SCREENS ADDED:
- User profile screen (/profile)
- Login and register screens

FILES TO CHECK:
- src/components/Authentication/SignIn.jsx
- src/components/Authentication/SignUp.jsx
- src/pages/profile/Profile.jsx
- src/pages/profile/ProfileRightBar.jsx
- src/pages/profile/profile.css

### CODE REVIEW

File: src/components/Buttons/Button.jsx [hunk 1]
```diff
@@ -8,9 +8,9 @@ const Button = (props) => {
       onClick={onClick}
       className={`margin-trb-16 btn btn-filled-primary ${buttonStyle}`}
     >
-      {icon ? icon : null}
-      {" "}
       {buttonText}
+      {" "}
+      {icon ? icon : null}
```
Comment (itsmebipul00) [inline src/components/Buttons/Button.jsx:13]: Suggestion: && would work fine here

File: src/components/LeftSidebar/left-sidebar.css [hunk 2]
```diff
@@ -1,6 +1,13 @@
 .left-sidebar-container{
   flex: 25%;
-  height: 90vh;
+  /* height: 90vh; */
```
Comment (itsmebipul00) [inline src/components/LeftSidebar/left-sidebar.css:3]: remove commented code

File: src/components/LeftSidebar/left-sidebar.css [hunk 3]
```diff
@@ -12,9 +19,9 @@
 }
 
 .sidebar-list-item:hover{
-  background-color: var(--grey-background-color);
+  background-color: var(--border-color);
   transform: scale(1.2);
-  border-radius: 1rem;
+  /* border-radius:/ 1rem; */
```
Comment (itsmebipul00) [inline src/components/LeftSidebar/left-sidebar.css]: here too

File: src/pages/login/Login.jsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import "./login.css";
+import dummy from '../../components/Header/dummy_profile_img.png'
+import { SignIn } from "../../components";
+// import {Login} from '../../components'
```
Comment (itsmebipul00) [inline src/pages/login/Login.jsx]: here too

**Review by itsmebipul00 — COMMENTED** [review]
Good to merge Nice Work

File: src/components/Feed/feed.css [hunk 5]
```diff
@@ -1,6 +1,6 @@
 .feed-container{
   flex: 50%;
-  height: 90vh;
+  /* height: 90vh; */
 }
 
 .feed-wrapper{
```
Comment (sudipkundu999) [inline src/components/Feed/feed.css:7]: give a max-width to this wrapper.
In larger screen it becomes very big
Attaching a screenshot for your reference
abhishek

File: src/components/LeftSidebar/left-sidebar.css [hunk 6]
```diff
@@ -12,9 +19,9 @@
 }
 
 .sidebar-list-item:hover{
```
Comment (sudipkundu999) [inline src/components/LeftSidebar/left-sidebar.css:21]: On hovering over this, based on the screen size the hover effect works differently,
attaching a screenshot for your reference
aside_abhishek

**Review by sudipkundu999 — APPROVED** [review]
The code looks good. 
Just fix the UI bugs and you're good to merge

**Review by itsmebipul00 — APPROVED** [review]

File: src/components/LeftSidebar/left-sidebar.css (hunk 6 above)
Comment (abhishektanwar) [reply src/components/LeftSidebar/left-sidebar.css:21]: fixed

File: src/components/Feed/feed.css (hunk 5 above)
Comment (abhishektanwar) [reply src/components/Feed/feed.css:7]: fixed

File: src/pages/login/Login.jsx (hunk 4 above)
Comment (abhishektanwar) [reply src/pages/login/Login.jsx]: removed

File: src/components/LeftSidebar/left-sidebar.css (hunk 3 above)
Comment (abhishektanwar) [reply src/components/LeftSidebar/left-sidebar.css]: removed

**Pull request merged**